core-bundle/src/Resources/contao/languages/en/tl_page.xlf

```diff
@@ -171,13 +171,19 @@
         <source>Set cache timeout values for the page and its subpages.</source>
       </trans-unit>
       <trans-unit id="tl_page.cache.0">
-        <source>Server cache timeout</source>
+        <source>Shared cache (server cache) timeout</source>
       </trans-unit>
       <trans-unit id="tl_page.cache.1">
         <source>The number of seconds after which the page should no longer be considered fresh by shared caches.</source>
       </trans-unit>
+      <trans-unit id="tl_page.alwaysLoadFromCache.0">
+        <source>Always load from shared cache (server cache)</source>
+      </trans-unit>
+      <trans-unit id="tl_page.alwaysLoadFromCache.1">
+        <source>Always load this page from the shared cache, even if a member is logged in. (Note that you can no longer personalize the page for logged in members in this case.)</source>
```

Comment:
No brackets needed imho

Reply:
True, updated in 871b9b13373b40b7bba0f70efc4f2ded10b52895.